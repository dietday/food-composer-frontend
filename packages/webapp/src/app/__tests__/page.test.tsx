import {render} from '@testing-library/react'
import Home from '../page'

test('test', () => {
  const {container} = render(<Home />)
  expect(container).toMatchSnapshot();
});
