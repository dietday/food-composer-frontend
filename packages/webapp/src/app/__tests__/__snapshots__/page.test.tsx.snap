// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test 1`] = `
<div>
  <p>
    0
  </p>
  <button
    class="btn"
  >
    Increment!!!
  </button>
  <button
    class="btn"
  >
    Decrement!!!
  </button>
</div>
`;
